src/framework/uicomponents/qml/MuseScore/UiComponents/FlatButton.qml

```diff
@@ -57,8 +57,8 @@ FocusScope {
 
     objectName: root.text
 
-    height: contentLoader.item.height + 14
-    width: (Boolean(text) ? Math.max(contentLoader.item.width + 32, prv.isVertical ? 132 : 0) : contentLoader.item.width + 16)
+    height: contentLoader.item.height + 11
+    width: (Boolean(text) ? Math.max(contentLoader.item.width + 32, prv.isVertical ? 132 : 0) : contentLoader.item.width + 12)
```

Comment:
I couldn't find the cases when Boolean(text) was used here, or when prv.isVertical is true, so I didn't touch that side of things.
NB I really had no idea what I was doing here, I can't use Qt Creator and rebuilding with VS when you've only modified a qml file isn't 100% reliable (and if there are any errors in the qml file you only find out because MuseScore won't start up).